include/modem/location.h

```diff
@@ -494,20 +494,39 @@ struct location_config {
  */
 typedef void (*location_event_handler_t)(const struct location_event_data *event_data);
 
+/**
+ * Register handler for location events.
```

Comment:
Is it clear enough that `location_handler_register()` is used to add an event handler and `location_handler_deregister()` to remove an event handler, and that there can be multiple event handlers? It's not said explicitly.

Reply:
Adding:
`Multiple event handlers can be registered.`